packages/components/src/Enumeration/Items/Item/ItemEdit.component.js

```diff
@@ -135,6 +143,10 @@ class ItemEdit extends React.Component {
 				<div className={itemEditActionsClasses()}>
 					{editActions.map((action, index) => this.getAction(action, index))}
 				</div>
+				{
+					this.props.item.error ?
```

Comment:
use rather this.props.item.error && <div className={itemErrorClasses()}>{this.props.item.error}</div>